Models/Flip.cs

```diff
@@ -0,0 +1,25 @@
+
+using System;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+
+namespace Coflnet.Sky.SkyAuctionTracker.Models
+{
+    [DataContract]
+    public class Flip
+    {
+        [IgnoreDataMember]
+        [JsonIgnore]
+        public int Id { get; set; }
+
+        [DataMember(Name = "auctionUUID")]
+        public string AuctionUUID { get; set; }
+        [DataMember(Name = "targetPrice")]
+        public int TargetPrice { get; set; }
+        [DataMember(Name = "finderType")]
+        public string FinderType { get; set; }
```

Comment:
this should NOT be a string because this reflects a text or varchar in the db that takes a lot of space.